integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,9 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// We set memory limit and disabled swap memory limit, this means the
```

Comment:
I think this simple test doesn't really need a comment (golang src doesn't always follow golint in tests as well). Sorry, I don't like it, but it could be just my taste :)